src/components/academy/grading/index.tsx

```diff
@@ -242,6 +284,34 @@ class Grading extends React.Component<IGradingProps, State> {
     }
     this.gridApi.exportDataAsCsv({ allColumns: true });
   };
+
+  private handleUnsubmitAlertClose = () => this.setState({ unsubmitAlert: null });
+
+  private unsubmitButton = (props: GradingNavLinkProps) => {
+    const handleConfirmUnsubmit = () => {
+      this.props.handleUnsubmitSubmission(props.data.submissionId, this.props.gradingOverviews!);
+      this.setState({ unsubmitAlert: null });
+    };
+
+    const unsubmitAlert = (
+      <Alert
+        canEscapeKeyCancel={true}
+        canOutsideClickCancel={true}
+        cancelButtonText="Cancel"
+        className="unsubmit-alert alert"
+        intent={Intent.DANGER}
+        onConfirm={handleConfirmUnsubmit}
+        isOpen={true}
+        onCancel={this.handleUnsubmitAlertClose}
+      >
+        <p>Are you sure you want to unsubmit this submission?</p>
+      </Alert>
+    );
+
+    return controlButton('', IconNames.ARROW_LEFT, () => this.setState({ unsubmitAlert }), {
+      fullWidth: true
```

Comment:
What is this doing?

Reply:
`fullWidth` is an option for the `controlButton` component, setting the button's width to 100% i believe